docs/releases/5.0.8.txt

```diff
@@ -11,3 +11,5 @@ Bugfixes
 
 * Added missing validation for ``UniqueConstraint(nulls_distinct=False)`` when
   using ``*expressions`` (:ticket:`35594`).
+* Added support for validation of model constraints which use a ``GeneratedField``
+  (:ticket:`35575`).
```

Comment:
As this ticket has been classed as a new feature, we won't backport to 5.0 (which only accepts bug fixes)
Can you instead add a release note to [Django 5.2 in the Models section](https://docs.djangoproject.com/en/dev/releases/5.2/#models)?